### ISSUE TITLE
Automatic Display Answer Time per deck 

### ISSUE THREAD

**Issue by gajewsk2** [ankidroid/Anki-Android#3838]
As a user I want to be able to set the time to automatically show the answer and go to the next card per deck. 

The Automatic display answer's Time to show next answer and Time to show answer setting are currently set in the Reviewing settings menu, forcing then to be one time per collection. It would be really convenient to save that per deck as not all cards take the same time to review, eg. sentence decks,  vocab decks, video decks. 

Any way we can move this to the deck options settings?

**Comment by eginhard** [comment]
This is a feature that would have to be implemented in Anki Desktop first before we could adopt it. You can suggest it in the main Anki forum: https://anki.tenderapp.com/discussions

**Comment by gajewsk2** [comment]
Hmm, but automatic displaying the answer doesn't even exist in Anki Desktop. So why would it have to exist there first?

**Comment by timrae** [comment]
It's just that all of the deck options are stored in the database and synced / completely consistent with Anki desktop. What exactly are you using the automatic show answer feature for? Using profiles might be a reasonable solution for you, although they're not officially supported yet (#2545)

**Comment by gajewsk2** [comment]
Oh really? I was afraid of that. Haven't gotten a chance to look at the code, but would having an extra field break Anki desktop? 

My personal use case: I'm using the 20s time per sentence card in one deck and don't want to automatically display the answer for video cards in another deck.

**Comment by timrae** [comment]
No it probably wouldn't break Anki desktop. If you're willing to implement it yourself then I guess we could add a new category "Advanced" to the deck options where we could put some AnkiDroid specific options. I'll warn you in advance though, the deck options code is a real mess, it's been in need of a good refactoring for a while.

**Comment by gajewsk2** [comment]
That sounds appropriate. So this is something we could actually get released without Anki desktop? If that's the case, I'll start poking around when I get a chance.

**Comment by timrae** [comment]
Yes, as long as we can make it clear which deck options are AnkiDroid specific,  and the code doesn't end up more tangled than it already is I don't have a problem with it :-D

**Comment by hssm** [comment]
You're basically free to shove whatever you want into the deck or collection configuration. Add-ons on the desktop have been doing that forever which allows them to sync their settings.
